generators/app/__snapshots__/generator.spec.ts.snap

```diff
@@ -255,6 +259,22 @@ exports[`generator - app with default config should match snapshot 1`] = `
   "camelizedBaseName": "jhipster",
   "capitalizedBaseName": "Jhipster",
   "caret": undefined,
+  "ciCd": undefined,
+  "ciCdAny": false,
+  "ciCdAzure": false,
+  "ciCdCircle": false,
+  "ciCdGithub": false,
```

Comment:
Should Hub be capitalized like it is in GitHub?

Reply:
Was like this previously, didn't changed anything here